translations/texts/objects/apex/apexcoolbookcase/apexcoolbookcase.object.json

```
@@ -92,7 +92,7 @@
     },
     "Texts": {
       "Eng": "Perhaps there is something to learn from the Apex.",
-      "Rus": ""
+      "Rus": "Возможно, есть чему поучиться у Апексов."
```

```suggestion
      "Rus": "Возможно, у Апексов есть чему поучиться."
```
Такой порядок слов более естественный для русского языка